sources/platform/actors/development/actor_definition/docker.md

```diff
@@ -0,0 +1,108 @@
+
+:::
+
+### Node.js base images
+
+These images come with Node.js (versions `16`, `18`, `20`, or `22`) the [Apify SDK for JavaScript](/sdk/js), and [Crawlee](https://crawlee.dev/) preinstalled. The `latest` tag corresponds to the latest LTS version of Node.js.
+
+| Image | Description |
+| ----- | ----------- |
+| [`actor-node`](https://hub.docker.com/r/apify/actor-node/) | Slim Alpine Linux image with only essential tools. Does not include headless browsers. |
+| [`actor-node-puppeteer-chrome`](https://hub.docker.com/r/apify/actor-node-puppeteer-chrome/) | Debian image with Chromium, Google Chrome, and the [`puppeteer`](https://github.com/puppeteer/puppeteer) library. |
+| [`actor-node-playwright-chrome`](https://hub.docker.com/r/apify/actor-node-playwright-chrome/) | Debian image with Chromium, Google Chrome, and the [`playwright`](https://github.com/microsoft/playwright) library. |
+| [`actor-node-playwright-firefox`](https://hub.docker.com/r/apify/actor-node-playwright-firefox/) | Debian image with Firefox and the [`playwright`](https://github.com/microsoft/playwright) library . |
+| [`actor-node-playwright-webkit`](https://hub.docker.com/r/apify/actor-node-playwright-webkit/) | Ubuntu image with WebKit and the [`playwright`](https://github.com/microsoft/playwright) library. |
+| [`actor-node-playwright`](https://hub.docker.com/r/apify/actor-node-playwright/) | Ubuntu image with [`playwright`](https://github.com/microsoft/playwright) and all its browsers (Chromium, Google Chrome, Firefox, WebKit). |
+
+See the [Docker image guide](/sdk/js/docs/guides/docker-images) for more details.
+
+### Python base images
+
+These images come with Python (version `3.8`, `3.9`, `3.10`, `3.11`, or `3.12`) and the [Apify SDK for Python](/sdk/python) preinstalled. The `latest` tag corresponds to the latest Python 3 version supported by the Apify SDK.
+
+| Image | Description |
+| ----- | ----------- |
+| [`actor-python`](https://hub.docker.com/r/apify/actor-python) | Slim Debian image with only the Apify SDK for Python. Does not include headless browsers. |
+| [`actor-python-playwright`](https://hub.docker.com/r/apify/actor-python-playwright) | Debian image with [`playwright`](https://github.com/microsoft/playwright) and all its browsers. |
+| [`actor-python-selenium`](https://hub.docker.com/r/apify/actor-python-selenium) | Debian image with [`selenium`](https://github.com/seleniumhq/selenium), Google Chrome, and [ChromeDriver](https://chromedriver.chromium.org/). |
+
+## Custom Dockerfile
+
+Apify uses Docker to build and run Actors. If you create an Actor from a template, it already contains an optimized `Dockerfile` for the given use case.
+
+To use a custom `Dockerfile`, you can either:
+
+* Reference it from the `dockerfile` field in `.actor/actor.json`,
+* Store it in `.actor/Dockerfile` or `Dockerfile` in the root directory (searched in this order of preference).
+
+If no `Dockerfile` is provided, the system uses the following default:
```

Comment:
@fnesveda I feel like this default docker image should be hosted somewhere if possible, instead of having it mentioned here. Is that possible? Can it be something that's maybe shown in apify-docker-images?

Reply:
Oh, sorry, I somehow missed this mention. Anyway - it is definitely possible, but is it really necessary? It changes like once a year, with a new Node version, so changing it in two places is not really that much extra work. If we had it in [`apify-actor-docker`](https://github.com/apify/apify-actor-docker), we'd have to fetch it during worker startup, and deal with network issues, GitHub outages, we'd have to take care not to move it to a different folder accidentally... I don't think it's worth it.

Reply:
And it's used for only very very few Actors, since all templates, from which most Actors are created, have their own Dockerfile.